docs/ftldns/configfile.md

```diff
@@ -122,75 +151,75 @@ Specify path and filename of FTL's SQLite3 gravity database. This database conta
 
 Print information about ARP table processing: How long did parsing take, whether read MAC addresses are valid, and if the `macvendor.db` file exists.
 
-#### `DEBUG_REGEX=false|true` {#debug_regex data-toc-label='Debug REGEX'}
+#### `DEBUG_REGEX=false|true` {#debug_regex data-toc-label='REGEX'}
 
 Controls if *FTL*DNS should print extended details about regex matching into `pihole-FTL.log`.<br>
 Due to legacy reasons, we also support the following setting to be used for enabling the same functionality:<br>
 `REGEX_DEBUGMODE=false|true`
 Note that if one of them is set to `true`, the other one cannot be used to disable this setting again.<br>
 **[More details](regex/overview.md)**
 
-#### `DEBUG_API=false|true` {#debug_api data-toc-label='Debug Telnet'}
+#### `DEBUG_API=false|true` {#debug_api data-toc-label='Telnet'}
 
 Print extra debugging information during telnet API calls. Currently only used to send extra information when getting all queries.
 
-#### `DEBUG_OVERTIME=false|true` {#debug_overtime data-toc-label='Debug overTime'}
+#### `DEBUG_OVERTIME=false|true` {#debug_overtime data-toc-label='Over Time Data'}
 
 Print information about overTime memory operations, such as initializing or moving overTime slots.
 
-#### `DEBUG_EXTBLOCKED=false|true` {#debug_extblocked data-toc-label='Debug externally blocked'}
+#### `DEBUG_EXTBLOCKED=false|true` {#debug_extblocked data-toc-label='Externally blocked queries'}
 
 Print information about why FTL decided that certain queries were recognized as being externally blocked.
 
-#### `DEBUG_CAPS=false|true` {#debug_caps data-toc-label='Debug Linux capabilities'}
+#### `DEBUG_CAPS=false|true` {#debug_caps data-toc-label='Linux capabilities'}
 
 Print information about capabilities granted to the pihole-FTL process. The current capabilities are printed on receipt of `SIGHUP`, i.e., the current set of capabilities can be queried without restarting `pihole-FTL` (by setting `DEBUG_CAPS=true` and thereafter sending `killall -HUP pihole-FTL`).
 
-#### `DEBUG_DNSMASQ_LINES=false|true` {#debug_dnsmasq_lines data-toc-label='Debug dnsmasq lines'}
+#### `DEBUG_DNSMASQ_LINES=false|true` {#debug_dnsmasq_lines data-toc-label='Analyze dnsmasq log lines'}
 
 Print file and line causing a `dnsmasq` event into FTL's log files. This is handy to implement additional hooks missing from FTL.
 
-#### `DEBUG_VECTORS=false|true` {#debug_vectors data-toc-label='Debug FTL vectors'}
+#### `DEBUG_VECTORS=false|true` ([#725](https://github.com/pi-hole/FTL/pull/725)) {#debug_vectors data-toc-label='Vectors'}
 
 FTL uses dynamically allocated vectors for various tasks. This config option enables extensive debugging information such as information about allocation, referencing, deletion, and appending.
 
-#### `DEBUG_RESOLVER=false|true` {#debug_resolver data-toc-label='Debug FTL resolver'}
+#### `DEBUG_RESOLVER=false|true` ([#728](https://github.com/pi-hole/FTL/pull/728})) {#debug_resolver data-toc-label='Resolver details'}
```

Comment:
```suggestion
#### `DEBUG_RESOLVER=false|true` ([#728](https://github.com/pi-hole/FTL/pull/728)) {#debug_resolver data-toc-label='Resolver details'}
```

Is this a test?

Reply:
Rather a typo.

Reply:
Actually, rather an editor auto-completion fail.